app/assets/javascripts/discourse/app/models/bookmark.js

```diff
@@ -125,9 +125,21 @@ const Bookmark = RestModel.extend({
     ).capitalize();
   },
 
-  @discourseComputed("linked_post_number", "fancy_title", "topic_id")
-  topicLink(linked_post_number, fancy_title, id) {
-    return Topic.create({ id, fancy_title, linked_post_number });
+  @discourseComputed()
+  topicLink() {
```

Comment:
I found this property name confusing because I thought a link is being returned. Instead, it returns a Topic object.